INSTALL

```diff
@@ -115,17 +121,42 @@ Configuring NEST for Distributed Simulation with MPI
      -DMPI_C_COMPILER=myC_CompilerWrapper -Dwith-mpi=ON
   4. Sorry, you need to fix your MPI installation. 
 
-
 Disabling the Python Bindings (PyNEST)
 ======================================
 
 Please see also the file pynest/README.md in the documentation directory. If you
 are impatient, use 
 
-    --Dwith-python=OFF
+    -Dwith-python=OFF
 
 as argument to `cmake`.
 
+Python3 Binding (PyNEST)
+=========================
+
+To force a Python3-binding in a mixed Python2/3 environment pass
+
+    -Dwith-python=3
+
+as an argument to `cmake`.
+
+`cmake` usually autodetects your Python installation.
+In some cases `cmake` v2.8.12 might not be able to localize the Python interpreter
```

Comment:
Do we need the version number here, i.e. are we sure that the problem only happens with this exact version? I think the sentence is just as clear without the version.